Jenkinsfile

```diff
@@ -37,6 +37,20 @@ pipeline {
                   make -j 4'''
           }
         }
+        stage('Build CHIME kotekan with Clang') {
+          steps {
+            sh '''mkdir -p chime-build-clang
+                  cd chime-build-clang
+                  export CC=/usr/lib/clang-8.0/bin/clang
+                  export CXX=/usr/lib/clang-8.0/bin/clang++
```

Comment:
I'm tempted to just make this `CC=clang` and `CXX=clang++` then it will be upto the system to make sure the default clang version is a good one.